libraries/Microsoft.Bot.Builder.AI.QnA/StrictFiltersCompoundOperationType.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.AI.QnA
+{
+    /// <summary>
+    /// Compound Filter for Metadata Join Expression.
+    /// </summary>
+    public enum StrictFiltersCompoundOperationType
```

Comment:
MetadataJoinOperator

Reply:
this is already the name of the property in the REST API

---
In reply to: [486651004](https://github.com/microsoft/botbuilder-dotnet/pull/4446#discussion_r486651004) [](ancestors = 486651004)